app/policies/admin/report_policy.rb

```diff
@@ -4,9 +4,10 @@ module Admin
   # Policy for managing reports
   class ReportPolicy < ApplicationPolicy
     alias_rule :new?, :create?, to: :manage?
+    alias_rule :index?, to: :show?
 
-    def read?
```

Comment:
Ditch the hydra_etd baggage that used to have special rules for ActiveAdmin (which uses `read?` unlike anything else in our stack), now that we have a fresh new authorization adapter.